examples/stream.rs

```diff
@@ -2,7 +2,12 @@ use spinoff::{Spinner, Spinners, Streams};
 use std::{thread::sleep, time::Duration};
 
 fn main() {
-    let sp = Spinner::new_with_stream(Spinners::Aesthetic, "Loading in stderr...", None, Streams::Stderr);
```

Comment:
For example, this line has `106` characters, so it still gets re-aligned.
I think a max width exceeding 100 is not ideal, so some of the realignments are inescapable. 
What do you think?

Reply:
It's fine if it gets re-aligned in files like these, which are just examples. It's just a bit more unreadable when it's in the actual internal code itself.

Reply:
Yeah, I guess we don't need to do the formatting stuff.